Lib/distutils/command/upload.py

```diff
@@ -159,7 +159,7 @@ def upload_file(self, command, pyversion, filename):
                 body.write(title.encode('utf-8'))
                 body.write(b"\r\n\r\n")
                 body.write(value)
-                if value and value[-1:] == b'\r':
+                if value and (value[-1:] == b'\r') and (key != 'content'):
                     body.write(b'\n')  # write an extra newline (lurve Macs)
```

Comment:
I’m wondering if this if block is even needed.  Only Mac OS 9 used `\n` as end of line character, right?
We could delete both of these lines.

Reply:
Ah, I agree! (I alluded to this above)

Shall I change this PR to do that? It would save having to add the test case.

Reply:
Keep the test case!

Reply:
OK, I've zapped the newline correction. The test is still there, but a little odd - "make sure we don't append `'\n'` to anything," so I added an explanatory comment.